frontend/packages/core/src/NPS/feedback.tsx

```diff
@@ -185,21 +204,42 @@ const NPSFeedback = (opts: FeedbackOptions) => {
         container
         direction="row"
         alignItems="center"
-        justify="center"
-        style={{ padding: "16px" }}
+        style={{ padding: wizardOrigin ? "16px" : "32px" }}
       >
-        <MuiGrid item xs={6}>
-          <Typography variant="subtitle3">{survey.prompt}</Typography>
+        <MuiGrid item xs>
+          <Typography variant={wizardOrigin ? "subtitle3" : "subtitle2"}>
+            {survey.prompt}
+          </Typography>
         </MuiGrid>
-        <MuiGrid item xs={6} style={{ display: "flex", justifyContent: "space-around" }}>
-          <EmojiRatings ratings={survey.ratingLabels} setRating={setSelected} />
+        <MuiGrid
+          item
+          xs={wizardOrigin ? 6 : 12}
+          style={{ margin: "8px 0px", display: "flex", justifyContent: "space-around" }}
```

Comment:
out of curiosity, is the `margin` needed? noticed that it makes the Wizard placement height look larger than before